Functions.Templates/Templates/SasToken-CSharp/metadata.json

```diff
@@ -1,11 +1,12 @@
 {
     "defaultFunctionName": "SasTokenCSharp",
-    "description": "A C# function that generates a SAS token for Azure Storage for a given container and blob name",
+    "description": "$SasToken_description",
     "name": "SAS Token Generator",
```

Comment:
token should be lower case?

Reply:
This is mapping to a resource. It is lower case in that resource description

Reply:
I think he means in the name property. That should be sentence-case in general: "SAS token generator"